packages/kolibri-tools/package.json

```
@@ -49,11 +50,13 @@
     "file-loader": "^2.0.0",
     "glob": "^7.1.2",
     "htmlhint": "^0.9.13",
+    "ini": "^2.0.0",
     "intl-messageformat-parser": "^1.3.0",
     "jest": "^24.5.0",
     "jest-serializer-vue": "^2.0.2",
     "json-loader": "^0.5.4",
     "launch-editor-middleware": "^2.2.1",
+    "lodash": "^4.17.21",
```

Doing this would avoid adding yet another copy of lodash:

```suggestion
    "lodash": "^4.17.20",
```

ref: https://github.com/learningequality/kolibri/pull/8224/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de